### ISSUE TITLE
1

### ISSUE THREAD

**Issue by ghost** [tczhangzhi/PyTorch-ESCGAN#1]

**Comment by tczhangzhi** [comment]
Thank you for your interest. Sorry for delaying the new version due to the tight schedule in recent months.

I hope I can complete the vision and provide a plug-and-play tool.

**Comment by Hanshuzheng149** [comment]
想问一下您，项目的执行顺序是什么？
1.train_evaluator.py(在没有数据增强时，测试准确率)
2.train_augment.py(进行数据增强)
3.train_gan.py(训练模型)
4.train_evaluator.py（数据增强之后进行测试）
是这样吗？谢谢！

**Comment by tczhangzhi** [comment]
Yes. Please refer to Sec. Model Optimization for `train_gan.py`. Please refer to Sec. Classifier Optimization for `train_augment.py`.
